Sources/Fuzzilli/Minimization/BlockReducer.swift

```diff
@@ -168,9 +170,10 @@ struct BlockReducer: Reducer {
         //             const v17 = Math(v16,v16);
         //         }
         //      } catch {
+        //      } finally {
         //      }
         //
-        if candidates.count == 4 {
+        if (candidates.count - tryCatch.numBlocks) == 0 {
             candidates.removeLast()
         }
 
```

Comment:
I think the following comment isn't correct, so while you are at it, maybe change it to "Remove all instructions in the body of the try block"? Also (again not your code), I think line 182 can just become `if !(code[i].op is Nop) {` (remove `tryCatch.`)